www/test/functional/page-objects/report.js

```diff
@@ -1,17 +1,12 @@
  var Report = function () {
 
   this.fillAddress = function (content) {
-    return element(by.model('report.address')).sendKeys(content);
+    element(by.model('report.address')).sendKeys(content);
+    return this;
   };
 
   this.getAddress = function() {
     return element(by.model('report.address')).getAttribute('value');
-  }
-
-  this.fillDate = function (date) {
-    return element(by.className('date')).sendKeys(date.getMonth(),date.getDate(),date.getYear(),
-    protractor.Key.TAB,date.getHours(),date.getMinutes(),
-    protractor.Key.TAB, "PM");
```

Comment:
Actually the method in being deleted here (sorry). We don't need this method anymore, since we have another one that deals with `moment`.